blocks/init/src/Blocks/custom/columns/manifest.json

```diff
@@ -142,5 +142,23 @@
 			"max": 100,
 			"step": 10
 		}
+	},
+	"variables": {
+		"columnsVerticalSpacing": [
+			{
+				"inverse": true,
+				"variable": {
+					"columns-vertical-spacing": "calc((%value% / 16) * 1rem)"
+				}
+			}
+		],
+		"columnsGutter": [
```

Comment:
just a suggestion but potentially to use name such as vertical and horizontal spacing since they both represent gap(column and row)😄